README.md

```diff
@@ -43,14 +51,39 @@ Reference: <https://firebase.google.com/docs/flutter/setup?platform=web>
    firebase login
    ```
 
-4. Install FlutterFire
+4. Configure with your firebase project 
+    ```bash
+   firebase init
+   ```
+
+5. Install FlutterFire
 
    ```bash
    dart pub global activate flutterfire_cli
    ```
 
-5. Configure it using this command in the Project
+6. Configure it using this command in the Project
 
    ```bash
    flutterfire configure
    ```
+
+7. Disable the firebase emulator dev environment 
+   
+**disable the following lines of codes in `lib/main.dart` or build in production mode**
+```dart
+  if (kDebugMode) {
+    try {
+      const host = '0.0.0.0';
+      FirebaseStorage.instance.useStorageEmulator(host, 9199);
+      FirebaseFirestore.instance.useFirestoreEmulator(host, 8080);
+    } catch (e) {
+      // ignore: avoid_print
+      print(e);
+    }
+  }
+```
```

Comment:
This isn't required since there's the `kDebugMode`, it's not mandatory change. It's only required when the dev wants to test their debug build against to firebase server rather than the emulator.